TileFormats/Instanced3DModel/README.md

```diff
@@ -22,65 +22,193 @@ A tile is composed of a header section immediately followed by a body section.
+The `z` vector would map onto a `forward` vector, but it is omitted because it will always be the cross product of `right` and `up`.
+
+**Figure 2**: A box in the standard basis.
+
+![box standard basis](figures/box-standard-basis.png)
+
+**Figure 3**: A box transformed into a rotated basis.
 
-_TODO: create a separate Batch Table spec that b3dm, i3dm, etc. can reference, [#32](https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/32)?_
+![box rotated basis](figures/box-rotated-basis.png)
 
-The batch table is a `UTF-8` string containing JSON.  It immediately follows the header.  It can be extracted from the arraybuffer using the `TextDecoder` JavaScript API and transformed to a JavaScript object with `JSON.parse`.
+#### Oct-encoded Normal Vectors
 
-Each property in the object is an array with its length equal to `header.batchLength`.  Array elements can be any valid JSON data type, including objects and arrays.  Elements may be `null`.
+If `NORMAL_UP` and `NORMAL_RIGHT` are not defined for an instance, its orientation may be stored as oct-encoded normals in `NORMAL_UP_OCT32P` and `NORMAL_RIGHT_OCT32P`.
+These define `up` and `right` using the oct-encoding described in
+[*A Survey of Efficient Representations of Independent Unit Vectors* by Cigolle et al.](http://jcgt.org/published/0003/02/01/).
+An implementation for encoding and decoding these unit vectors can be found in Cesium's
+[AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js)
+module.
+
+### Instance Position
+
+`POSITION` defines the location for an instance before any tileset transforms are applied. 
+
+#### Quantized Positions
+
+If `POSITION` is not defined for an instance, its position may be stored in `POSITION_QUANTIZED` which defines the instance position relative to the quantized volume.
+If neither `POSITION` or `POSITION_QUANTIZED` are defined, the instance will not be created.
+
+A quantized volume is defined by `offset` and `scale` to map quantized positions into model space.
+
+**Figure 4**: A quantized volume based on `offset` and `scale`.
+
+![quantized volume](figures/quantized-volume.png)
+
+`offset` is stored in the global semantic `QUANTIZED_VOLUME_OFFSET`, and `scale` is stored in the global semantic `QUANTIZED_VOLUME_SCALE`.
+If those global semantics are not defined, `POSITION_QUANTIZED` cannot be used.
+
+Quantized positions can be mapped to model space using the formula:
+
+`POSITION = POSITION_QUANTIZED * QUANTIZED_VOLUME_SCALE + QUANTIZED_VOLUME_OFFSET`
+
+### Instance Scaling
+
+Scaling can be applied to instances using the `SCALE` and `SCALE_NON_UNIFORM` semantics.
+`SCALE` applies a uniform scale along all axes, and `SCALE_NON_UNIFORM` applies scaling to the `x`, `y`, and `z` axes independently.
+
+### Examples
+
+In these examples, the semantic values are shown as JSON arrays. This is done to make the examples more human readable, and is still a valid feature table.
```

Comment:
The new approach with the code example is good (actually really good for those doing writers in Node.js).  Update this paragraph since it is out-of-date.